src/main/java/io/confluent/connect/jdbc/gp/GpDataIngestionService.java

```diff
@@ -139,24 +139,58 @@ public void ingest(List<SinkRecord> records) {
 
 
         }
+        if (config.printDebugLogs) {
+            log.info("excluded columns list for update " + config.updateExcludeColumns);
 
+            log.info("excluded columns list for insert " + config.insertExcludeColumns);
+        }
 
         // add all columns except the updateExcludeColumns to updateColumnsList, excluded columns may have fully qualified names like tablename.columnname
-        List<String> excludedColumns = config.updateExcludeColumns.stream().filter(
-                column -> column.contains(".") ? column.split("\\.")[0].trim().equals(tableName) : true
-        ).collect(Collectors.toList());
+        List<String> excludedColumns = config.updateExcludeColumns.stream()
+                .map(String::trim)
```

Comment:
Looks like this statement will exclude all columns not having a table name with them

Reply:
Yes, the statement was excluding all the columns not having a table name. I have fixed this. Please review.